### DISCUSSION TITLE
Hide things on launcher

### DISCUSSION THREAD

**Discussion by Wassax7** [SKCraft/Launcher#417] [I need help!]
Hi,

I'd like to know if there is a way to first, hide the left panel for selecting mods. I just need one pack to download so there is no need for selection.

And is it possible to hide the console or don't open it at all at launch?

Your launcher is really great thank you for work and answer.

**Comment by hedgehog1029** [comment]
There's no easy config option for either. You are of course free to modify the code in your local copy.

On the subject of single-pack downloads; I have been considering making a "mini-launcher" variant that is basically a single executable that only downloads & launches a single pack, with no news feed or anything. So that might be an option in future.
